package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "user-boilerplate",
-  "version": "0.3.0",
+  "version": "1.0.0",
   "description": "User Boilerplate",
   "main": "lib/index.js",
   "scripts": {
     "start": "node server",
-    "test": "lab -a code -L",
-    "lint": "lab -dL"
+    "test": "lab -a code -I '__core-js_shared__' -L",
```

Comment:
This global kills me... if we solve babel's `__core-js_shared__` it would probably make us famous :P